### PR TITLE
Show a different message when we cant connect to the languages update site

### PR DESCRIPTION
**Pull request by zero-24**
Pull Request for Issue #25987 

### Summary of Changes

Check whether we have a update site and if not sow the current message; Create a new message when the connect fail.

@brianteeman please suggest an message and let me know whether you want to include the update server url or not :)

##### First test
install on localhost
disconect from the internet
try to install new languages
see new error message

##### second test

install core
clear the #__updates_sites table using phpmyadmin
try to install new languages
see the current error message
rebuild the update servers as suggested
error message should be gone.

### Expected result

new error message and the old only on broken sites

### Actual result

always the old message

### Documentation Changes Required

none.

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_installer.ini [hunk 1]
```diff
@@ -120,6 +120,7 @@ COM_INSTALLER_MSG_DISCOVER_INSTALLSUCCESSFUL="Discover install successful."
 COM_INSTALLER_MSG_DISCOVER_NOEXTENSION="<strong>No extensions have been discovered.</strong> Select Discover to find new extensions that might be available for install."
 COM_INSTALLER_MSG_DISCOVER_NOEXTENSIONSELECTED="No extension selected."
 COM_INSTALLER_MSG_DISCOVER_PURGEDDISCOVEREDEXTENSIONS="Cleared discovered extensions."
+COM_INSTALLER_MSG_ERROR_CANT_CONNECT_TO_UPDATESERVER="Can't connect to %s please check your internet connection."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_installer.ini]: this isnt correct as its not the users internet connection. 

Leave it with me and I will come up with something - just got off a long flight

**Comment by brianteeman** [comment]
This PR is for J3 - I reported it for J4

File: administrator/language/en-GB/en-GB.com_installer.ini (hunk 1 above)
Comment (zero-24) [reply administrator/language/en-GB/en-GB.com_installer.ini]: > Leave it with me and I will come up with something - just got off a long flight

Great Thanks. 👍 safe travels :)

Comment (zero-24) [reply administrator/language/en-GB/en-GB.com_installer.ini]: @brianteeman would it be better to write `Can't connect to %s please check the internet connection.`

Comment (brianteeman) [reply administrator/language/en-GB/en-GB.com_installer.ini]: Sorry I forgot about this.

It is still not really correct as we are talking about the connection between your server and the joomla update server. The internet connection is obviously ok as you are connected to it in order to see that message. 

I would just go for 
`Can't connect to %s`

Comment (zero-24) [reply administrator/language/en-GB/en-GB.com_installer.ini]: pushed. thanks.

File: administrator/components/com_installer/models/languages.php [hunk 2]
```diff
@@ -113,6 +113,14 @@ protected function getLanguages()
 	{
 		$updateSite = $this->getUpdateSite();
 
+		// Check whether the the updateserver is found
```
Comment (Quy) [inline administrator/components/com_installer/models/languages.php]:
```suggestion
		// Check whether the updateserver is found
```

**Comment by SharkyKZ** [comment]
I have tested this item :white_check_mark: successfully on c68124a3dae31e215feae7fefc869aeaa431395a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25996">issues.joomla.org/tracker/joomla-cms/25996</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 03e6a6b4e75012294ffb7d3b4f0d9158be8aa100<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25996">issues.joomla.org/tracker/joomla-cms/25996</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25996">issues.joomla.org/tracker/joomla-cms/25996</a>.</sub>

**Comment by HLeithner** [comment]
Thanks

**Pull request merged**